### PR TITLE
Fix some activity popup SQL (#53) and chat (#20)

### LINKED ISSUE #20: [Activity] chat doesn't have username colors
screen:

### LINKED ISSUE #53: Fix SQL injections
I spotted some SQL injections in the Activity plugin.

### PR DESCRIPTION
**Pull request by vendethiel**
Fixes #53 (there might be some, still, not sure)
Fixes #20 (instead of parsing the chat everytime we want to access it, parse messages as we add them)

### CODE REVIEW

**Comment by vendethiel** [comment]
@MightyGorgon please merge

File: activity/ip_root/activity_popup.php [hunk 1]
```diff
@@ -215,19 +216,20 @@ function CheckReturnPath($id)
 		$bbcode->allow_smilies = $smilies_on;
 		$message = $bbcode->parse($message);
 		$message = prepare_message(trim($to_add), $html_on, $bbcode_on, $smilies_on);
+		$message = $bbcode->parse($message);
```
Comment (MightyGorgon) [inline activity/ip_root/activity_popup.php]: Why parsing again just one line below the other parse?

Comment (vendethiel) [inline activity/ip_root/activity_popup.php]: Uhm, probably a mistake.

File: activity/ip_root/activity_popup.php [hunk 2]
```diff
@@ -268,20 +269,8 @@ function CheckReturnPath($id)
 		$todays_chat = $db->sql_fetchrow($r);
 		$chat_session = $todays_chat['chat_text'];
 
-		$message = $chat_session;
```
Comment (MightyGorgon) [inline activity/ip_root/activity_popup.php]: Why you have removed this?

Comment (vendethiel) [inline activity/ip_root/activity_popup.php]: it is not needed to reparse chat

**Pull request closed without merging**